packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -96,4 +98,29 @@
                         Labels="@(ContainerLabel)"
                         ExposedPorts="@(ContainerPort)" />
     </Target>
+
+    <Target Name="PublishContainer" DependsOnTargets="$(PublishContainerDependsOn)"
+                                    Condition="'$(MSBuildRuntimeType)' == 'Full'">
```

Comment:
Unfortunately, conditions on tasks don't work this way--they're evaluated just before the task runs, so this target always wins and just won't run on Core, ever.

```
Target Name=PublishContainer Project=container-package-test.csproj
    Target "PublishContainer" skipped, due to false condition; ('$(MSBuildRuntimeType)' == 'Full') was evaluated as ('Core' == 'Full').
```

Reply:
Why would this target always win? Is this the only one defined in that case?